medicines/doc-index-updater/src/models.rs

```diff
@@ -83,22 +83,25 @@ impl Into<Document> for XMLDocument {
             author: self.author,
             products: self
                 .products
+                .product
                 .iter()
-                .map(move |active_substance| active_substance.name.clone())
+                .map(|product| product.to_string())
```

Comment:
This 3-line block is unnecessary. You're iterating just to return the initial thing.

Reply:
Yeah these blocks are no longer necessary, they were necessary when we weren't already deserializing to `Vec<String>`.